site/en/r2/tutorials/generative/cyclegan.ipynb

```diff
@@ -84,8 +112,8 @@
         "\n",
         "This opens up the possibility to do a lot of interesting tasks like photo-enhancement, image colorization, style transfer, etc. All you need is the source and the target dataset (which is simply a directory of images).\n",
         "\n",
-        "![Output Image 1](images/horse2zebra_1.png)\n",
-        "![Output Image 2](images/horse2zebra_2.png)"
+        "![Output Image 1](https://github.com/junghau/docs/blob/better-cyclegan-loss/site/en/r2/tutorials/generative/images/horse2zebra_1.png?raw=1)\n",
```

Comment:
Undo change to image path.

If this doesn't work for some reason, then use the tensorflow/docs repo.

Reply:
Do you mean to revert back the image paths to original, as in the form `images/horse2zebra_1.png`?

Reply:
The original image paths (images/horse2zebra_1.png) works fine. Why the change?